_sass/_base.scss

```diff
@@ -6,3 +6,7 @@ body {
 img {
   max-width: 100%;
 }
+
+.this__is-a--test {
+  color: #000;
```

Comment:
Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.